src/components/gds-body/gds-body.tsx

```diff
@@ -0,0 +1,25 @@
+import { Component, h, Prop } from '@stencil/core'
+
+/**
+ * Paragraph element with different sizes.
+ */
+@Component({
+  tag: 'gds-body',
```

Comment:
Should this be `gds-paragraph` rather than `gds-body`?

Reply:
Yeah, I was thinking that, but ended up with body since paragraph is probably not going to be the sole purpose of this element. We could make another `gds-paragraph` element later on? 

Maybe this element could be just `gds-text` instead of `gds-body`... but at least the body implies that it's the body text, and not any other kind. 

So maybe let's keep it for now and change later if need be.

Reply:
As it outputs a `<p>` element where would this be used if it wasnt a paragraph? But yeah sounds good to iterate on it later

Reply:
Ah, that's right. I'll add gds-paragraph.